src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -103,11 +103,41 @@ bool BulletRigidObject::initializeScene(
   return true;
 }  // end BulletRigidObject::initializeScene
 
+// recursively create the convex mesh shapes and add them to the compound in a
+// flat manner by accumulating transformations down the tree
+void BulletRigidObject::constructBulletConvexCompoundFromMeshes(
+    std::unique_ptr<btCompoundShape>& bCompound,
+    Magnum::Matrix4& T,
+    const std::vector<assets::CollisionMeshData>& meshGroup,
+    const assets::MeshTransformNode& node) {
+  Magnum::Matrix4 cT = T * node.T_parent_local;
```

Comment:
Assuming that `T` is `T_world_parent` (seems like it is from the doc string), then `cT` is `T_world_local`?